routine_butler/main.py

```diff
@@ -25,67 +35,77 @@ def set_colors():
     )
 
 
-class RoutineButler:
-    def __init__(self, repository: Repository, user: Optional[User] = None):
-        self.repository = repository
-        self.user = user
-        logger.debug(self.user)
-        self.main_frame = ui.element("div")
 
-        if user is None:
-            with self.main_frame:
-                self.login()
-        else:
-            # set repository current_username attribute
-            self.repository.current_username = user.username
-            # instantiate main gui without login
-            with self.main_frame:
-                self.main_gui()
+class DBMiddleware:
```

Comment:
I think it could be useful for if me you added a few brief doc strings (to briefly describe the purpose/function) for some of these additions. Just one or two sentences is fine--not full-on documentation.

Reply:
of course... but I was planning to explain in a meeting.